docs/website/docs/user-guides/deploy/_running_deploy.mdx

```diff
@@ -0,0 +1,42 @@
+import CodeBlock from '@theme/CodeBlock';
+import Version from '../../_version.mdx';
+
+Now we're ready to run `odo deploy`:
+
+```console
+odo deploy
+```
+
+<div>
+<CodeBlock language="console">
+{`\
+$ odo deploy
+  __
+ /  \\__     Deploying the application using `}{props.name}{` Devfile
+ \\__/  \\    Namespace: default`}<Version/>{`
+↪ Building & Pushing Container: MYUSERNAME/`}{props.name}{`-odo-example
```

Comment:
Since this is an example output, I think it makes sense to use your own username and replace `MYUSERNAME` with cdrage or something else.